requirements/edx/base.txt

```diff
@@ -136,6 +136,7 @@ freezegun==0.1.11
 lettuce==0.2.20
 mock-django==0.6.6
 mock==1.0.1
+moto==0.3.1
```

Comment:
@wedaly instead of allowing outdated dependencies to dictate which libraries we can use, I'd much prefer to upgrade dependencies where it makes sense. Can you upgrade `boto` in this pull request, so that we can use the most recent version of `moto`?